src/luascript.cpp

```diff
@@ -982,6 +982,16 @@ void LuaScriptInterface::pushOutfit(lua_State* L, const Outfit* outfit)
 	setMetatable(L, -1, "Outfit");
 }
 
+void LuaScriptInterface::pushMount(lua_State* L, const Mount* mount)
+{
+	lua_createtable(L, 0, 5);
+	setField(L, "name", mount->name);
+	setField(L, "speed", mount->speed);
+	setField(L, "clientId", mount->clientId);
```

Comment:
isnt clientId looktype?

Reply:
`<mount id="1" clientid="368" name="Widow Queen" speed="20" premium="yes" />`
yes?!

Reply:
I find it weird to be named clientid, but if its in xml then okay